### ISSUE TITLE
Case 5: various text issues

### ISSUE THREAD

**Issue by PixelSergey** [ScarletStudy/DGS1-3DS-Release#102]
Firstly, thank you for the game! I appreciate the immense amount of effort you put into this. Love you, Scarlet Study team.

Now, some issues:

----
1.
![image](https://user-images.githubusercontent.com/14542417/59549972-de949f00-8f6d-11e9-860f-650d780c21cd.png)
Phrased a bit weirdly, maybe rephrase to "...we declare this court to now be in session"/"...we now declare this court to be in session"/"...we now declare this court in session"?

----
2.
![image](https://user-images.githubusercontent.com/14542417/59549992-329f8380-8f6e-11e9-8828-f8d4080dfefa.png)
I think the intention here was for Tully to quote the scream, as in:

> ...we 'eard someone yell "'and over that gun!"

as opposed to

> ...we 'eard someone yell<b><i>!</i></b> "'and over that gun!"

----
3. 
![image](https://user-images.githubusercontent.com/14542417/59550006-772b1f00-8f6e-11e9-9dc9-0d7cba06bb53.png)
Although the word "enquiry" is used in Britain when referring to questions, I would say that the word "inquiry" would work better in this context, as he is referring to a formal investigation
(https://www.grammarly.com/blog/inquire-enquire/)

----
4. 
![image](https://user-images.githubusercontent.com/14542417/59550051-ddb03d00-8f6e-11e9-8fe7-b83885cb1adf.png)
Why does Naruhodou abbreviate cross-examination here? It is done literally nowhere else, and is quite jarring.

----
5. 
![image](https://user-images.githubusercontent.com/14542417/59550063-ff112900-8f6e-11e9-90b4-3b46de5c28b4.png)
This is not a typo, but rather a logical flaw. If you press the Tinpillars *about the milk company* before pressing *about the name of the company*, Ryuu will still say "You mean Rupert Milverton?" The fact that Chrogray's birth name was Milverton is only revealed after pressing *about the name*. Therefore, Ryuu cannot possibly know about the name Rupert Milverton yet, and the dialogue is incorrect.

This same dialogue also repeats if you press the statement *after* pressing about the name of the company (and, at that point, has no logical flaw). 

----
6.
![image](https://user-images.githubusercontent.com/14542417/59550169-7c896900-8f70-11e9-9cca-e1afb8c7d880.png)
![image](https://user-images.githubusercontent.com/14542417/59550172-801cf000-8f70-11e9-8e4d-eadf142d64bf.png)
This "Oof..." textbox is missing the speaker's name, Nemmy, in both the textbox and the backlog.

----
7. 
![image](https://user-images.githubusercontent.com/14542417/59550188-99be3780-8f70-11e9-97a6-5f485b9840cd.png)
![image](https://user-images.githubusercontent.com/14542417/59550190-9cb92800-8f70-11e9-9ed8-3a4d4e44dfce.png)
The periods are likely supposed to be inside the quotation marks.

----
8. 
![image](https://user-images.githubusercontent.com/14542417/59550199-b35f7f00-8f70-11e9-99fe-f1bce33e557e.png)
In the after-credits screen, this was left untranslated and got me curious. What does it say? Maybe add a translation to the bottom screen, if you want to leave the original graphic untouched?

----

Once again, thanks a lot for your work and your effort to fix bugs!

**Comment by wabberz** [comment]
About 8.: This is the title of the game and the big kanji means "The End". This is translated in the Android version, since we had to re-encode the credits video for that anyways, but for 3DS we figured it wasn't that important to justify re-encoding the video with a loss in quality and bloating the patch size...

**Comment by keksdee** [comment]
1. Will Consider™ this one
2. I think it works with the exclamation mark, but also works better with a comma...............but nah I'm gonna write something else lol
3. In this one, he's actually happy to respond to any questions, so I think the e is fine?
4. Fixed, thanks
5. Fixed, thanks (I think a consistency check went wrong)
6. That's how the JP was, too. I don't know if only one of them has mouth flaps here, but I figure both of them are ashamed (the judge is appalled about how they both ganged up on him)
7. Ugh jesus god who did this, I swear it wasn't me; fixed, thanks
